src/components/SubscriptAvatar.js

```diff
@@ -67,9 +67,13 @@ function SubscriptAvatar(props) {
                 accountID={lodashGet(props.secondaryAvatar, 'id', -1)}
                 icon={props.secondaryAvatar}
             >
-                <View style={props.size === CONST.AVATAR_SIZE.SMALL_NORMAL ? styles.flex1 : {}}>
+                <View
+                    style={[props.size === CONST.AVATAR_SIZE.SMALL_NORMAL ? styles.flex1 : {}, isSmall ? styles.secondAvatarSubscriptCompact : subscriptSyle]}
```

Comment:
Let's fix the old typo here
```suggestion
                    style={[props.size === CONST.AVATAR_SIZE.SMALL_NORMAL ? styles.flex1 : {}, isSmall ? styles.secondAvatarSubscriptCompact : subscriptStyle]}
```